app/grandchallenge/components/schemas.py

```diff
@@ -491,6 +491,67 @@
             "required": ["version", "type", "ellipses"],
             "additionalProperties": False,
         },
+        "3-point-angle-object": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"},
+                "type": {"const": "Three-point angle"},
+                "angle": {
+                    "type": "array",
+                    "items": {
+                        "type": "array",
+                        "items": {"type": "number"},
+                        "minItems": 3,
+                        "maxItems": 3,
+                    },
+                    "minItems": 3,
+                    "maxItems": 3,
+                },
+                "probability": {"type": "number", "minimum": 0, "maximum": 1},
+            },
+            "required": ["angle"],
+            "additionalProperties": False,
+        },
+        "3ANG": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"},
+                "type": {"const": "Three-point angle"},
+                "angle": {
+                    "type": "array",
+                    "items": {
+                        "type": "array",
+                        "items": {"type": "number"},
+                        "minItems": 3,
+                        "maxItems": 3,
+                    },
+                    "minItems": 3,
+                    "maxItems": 3,
+                },
+                "version": {"$ref": "#/definitions/version-object"},
+                "probability": {"type": "number", "minimum": 0, "maximum": 1},
+            },
+            "required": ["angle"],
+            "additionalProperties": False,
+        },
+        "M3AN": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"},
+                "type": {"enum": ["Multiple three-point angles"]},
+                "boxes": {
```

Comment:
Wouldn't `"angles"` be more appropriate than `"boxes"` here?

Reply:
Sorry yes! Too many updates necessary, this is a copy-paste mistake!